Zeeguu Reader/Pods/SwiftyJSON/README.md

```diff
@@ -0,0 +1,376 @@
+#SwiftyJSON [中文介绍](http://tangplin.github.io/swiftyjson/)
+
+[![Travis CI](https://travis-ci.org/SwiftyJSON/SwiftyJSON.svg?branch=master)](https://travis-ci.org/SwiftyJSON/SwiftyJSON)
+
+SwiftyJSON makes it easy to deal with JSON data in Swift.
```

Comment:
what's not clear to me is ... why do you have to commit the Reademe.md of this pod? Isn't the pod supposed to be pulled by cocoapods at build time?